### DISCUSSION TITLE
Many Core components which were available in v4 are not available in V5

### DISCUSSION THREAD

**Discussion by omscse** [vmware/clarity#5899] [Q&A] [answered]
Hello All,

Many Core components which were available in v4 are not available in V5. Ex. Cards, Datagrids. I am trying to use CDS core components with Angular frame work. Why many core components are taken off? What is the alternative?

https://clarity.design/core-components/
https://v4.clarity.design/documentation

Appreciate your help in this!

**Comment by mathisscott** [comment, marked as answer]
HI @omscse 
https://v4.clarity.design/documentation documents the Clarity components that were built with angular. Those can all still be found 👉 https://clarity.design/angular-components/

The core components are built as web components so they can be used in frameworks other than angular – or no framework at all.

We are still building out the library of core components. That is why you see a discrepancy. We are working as hard as we can to bridge that gap.

**Reply by omscse** [reply]
Thank you @manishmalik for quick response! It helps.

So I assume I can mix the v5 angular components and v5 core components in my project. Kindly suggest if there are any better ways.

Thanks,
Omprakash.

**Reply by mathisscott** [reply]
Yes, you can mix the components in the same app.
